posts/a-short-history-of-code-search.md

```diff
@@ -4,22 +4,22 @@ authorSlug: lisa-tagliaferri
 authorDisplayName: Lisa Tagliaferri
 tags: [tutorial, code search]
```

Comment:
I noticed no other tutorials had "code search" as a tag, I was not sure what other tags I could include here, or if we should add "code search" to other tutorials. Just something to point out

Reply:
`search` maybe? I can add that